srcpkgs/mspds/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'mspds'
+pkgname=mspds
+version=3.15.1.1
+revision=1
+archs="x86_64 i686"
```

Comment:
@ahesford seems like this actually needed after all :man_shrugging:

Reply:
I can't seem to see the results of the last CI run after your latest push, but `archs` exists to honor upstream functional restrictions, not as a workaround for avoiding cross-compilation challenges.

Reply:
fair, let me look up how to do cross compilation in void and work on that

Reply:
I tried doing cross compilation but I wasn't able to manage it. Take a look at this other package that also builds source from Texas Instruments and also limits to i686, x86_64
https://github.com/void-linux/void-packages/blob/master/srcpkgs/msp430-toolchain/template

Reply:
The template you reference specifies `x86_64* i686*`, which includes musl variants; your spec restricts the package to glibc only. Is there some reason this package can't build on musl?

I don't know why the other template doesn't support the cross-built archs; it would be nice to understand if that was a fundamental restriction or a CBA cop-out.

Reply:
`nocross=yes` would be more appropriate here, I think.

Reply:
:+1: